File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.py
class Solution:
    def reverseWords(self, s: str) -> str:
        lst = s.strip().split()
        low = 0
        high = len(lst)-1
        while(low<=high):
            temp = lst[low]
            lst[low] = lst[high]
            lst[high] = temp
            low+=1
            high-=1
        return " ".join(lst)

File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.java
class Solution {
    public String reverseWords(String s) {

        String words[] =s.trim().split("\\s+");
        int i=0;
        int n=words.length-1;
        while(i<=n){
            String word = words[i];
            words[i]=words[n];
            words[n]=word;
            i++;
            n--;
        }return String.join(" ",words);
    }
}